beacon-chain/p2p/handshake.go

```diff
@@ -172,7 +172,7 @@ func (s *Service) AddDisconnectionHandler(handler func(ctx context.Context, id p
 				s.host.ConnManager().Unprotect(conn.RemotePeer(), "protocol")
 				// Only log disconnections if we were fully connected.
 				if priorState == peers.PeerConnected {
-					log.WithField("active", len(s.peers.Active())).Info("Peer disconnected")
+					log.WithField("activePeers", len(s.peers.Active())).Info("Peer Disconnected")
```

Comment:
This punctuation was correct, please fix "Peer Connected" since that is improper.

```suggestion
					log.WithField("activePeers", len(s.peers.Active())).Info("Peer disconnected")
```